packages/flutter_tools/test/commands.shard/hermetic/build_linux_test.dart

```diff
@@ -611,6 +643,8 @@ set(BINARY_NAME "fizz_bar")
 
   testUsingContext('Performs code size analysis and sends analytics', () async {
     final BuildCommand command = BuildCommand(
+      flutterUsage: TestUsage(),
```

Comment:
can Usage and Platform overrides be removed from these tests?

Reply:
and if not, maybe we should be leaving TODO comments on what needs to happen before they can be removed.

Reply:
The `BuildLinuxCommand` should be migrated first to delete those, this PR is just for `BuildMacosCommand`

Reply:
ahhh, good point